docs/dev/helping-with-product-popularity-and-availability.MD

```diff
@@ -0,0 +1,64 @@
+# Ensuring User Scans Are Counted and Enhancing Availability Data with Open Prices
+
+This guide will help you ensure that your users' scans are properly counted, explain why it matters, and show how to go even further with the Open Prices project.
+
+---
+
+## Why should you make sure scans are counted?
+
+Properly reporting scans from your app provides several key benefits:
+
+- **Popularity Statistics:** Accurate scan counts help OFF understand which products are frequently scanned, indicating popularity and consumer interest.
+- **Allowing us to gauge our aggregated impact:** This allows to create aggregated scan counts with real impact on 1st and 3rd party users, measuring our social impact.
+- **Product Availability:** Frequent scans in a region suggest a product is available there, helping keep the database up to date.
+- **Data Quality:** More scans can signal which products need updating, corrections, or more data.
+- **Country of Sale:** Scan metadata can be used to deduce in which countries products are actually sold, especially when paired with price data.
+
+---
+
+## How to Ensure Scans from Your App Are Counted
+
+To count a scan, your app must send a request to the OFF API with a special HTTP header. This header **must contain the word "Scan"** (case-insensitive). This lets Open Food Facts distinguish user-initiated scans from other API calls (like lookups or background syncs).
+
+### Example: Including the Scan Header
+
+When making a request (e.g., fetching a product by barcode), add a custom header such as:
+
+TODO(): put actual query
+
+```http
+GET /api/v2/product/1234567890123.json HTTP/1.1
+Host: world.openfoodfacts.org
+User-Agent: MyAwesomeScanner/1.0
+X-OpenFoodFacts-Scan: Scan; app=MyAwesomeScanner; version=1.0
+```
+
+**Requirements:**
+- The header key can be any custom header (recommended: `X-OpenFoodFacts-Scan`).
```

Comment:
to check

Reply:
we probably want "scan" at the end, so that we don't have false positives for apps that are called "somethingScan" or "Scan-Something"